pkg/authorizer/server/types.go

```diff
@@ -1,21 +1,12 @@
 package server
 
-type TokenRequest struct {
-	APIVersion string `json:"api_version"`
-
-	AuthorizationToken string `json:"authorization_token"`
+type ClusterRegistration struct {
 	ClusterID          string `json:"cluster_id"`
+	AuthorizationToken string `json:"authorization_token"`
+	AccountID          string `json:"account_id"`
 }
 
-type TokenResponse struct {
-	APIVersion string `json:"api_version"`
-
-	Status  string `json:"status"`
-	Code    int    `json:"code"`
-	Reason  string `json:"reason"`
-	Message string `json:"message"`
-
-	AccountID string `json:"account_id"`
-
-	Labels map[string]string `json:"labels"`
+type Error struct {
```

Comment:
Similarly, this type is never used outside of this package. As long as the fields are exported, an unexported type can still be marshaled to JSON.